Altis_Life.Altis/stringtable.xml

```diff
@@ -4019,7 +4019,7 @@
             <Spanish>Estas a punto de morir de desidratación, busca algo de tomar rapido!</Spanish>
             <Russian></Russian>
             <German>Du bist am Verdursten! Du wirst sterben, wenn du nichts zum Trinken findest!</German>
-            <French>Vous êtes en train de mourir de soif, vous allez mourir très bientôt si vous ne buvez pas quelque chose.</French>
+            <French>Vous êtes en train de mourir de soif, vous allez mourir très bientôt si vous ne buvez pas.</French>
```

Comment:
Don't want to be rude, but it's a useless change. Letting the "quelque chose" isn't wrong at all.

Reply:
The "quelque chose" is not useful because if you drink it is necessarily "quelque chose".